### ISSUE TITLE
Tracker alignment Kinectv2

### ISSUE THREAD

**Issue by benybo** [simlrh/OSVR-Kinect#7]
Hello,
The position tracking of the KinectV2 is much better than the original one provided wih teh HDK2 OSVR.
The Kinect is at a great distance, But i can not adjust the forward/backward/right/left direction. If i advance, it goes to the left. If i retreat, it goes o the right. if i go right, it goes ahead. I followed some examples, use "osvr_reste_yaw.exe" which directs HDK2, bu not the direction of the Kinect. 
With OSVRTrackerView.exe, i have the movement of the hands in the right way, not the head. i use OSVR Rutime on windows10. Do you have the solution? Thank You. 
Sorry about my english, i am french... :)

**Comment by simlrh** [comment]
See #6

**Comment by benybo** [comment]
It's been 2 days I've been trying. But I have to do something wrong. but what? I copy / paste the line (osvr_reset_yaw.exe --path "/ com_osvr_Multiserver / OSVRHackerDevKitPrediction0 / semantic / hmd") into the "osvr_reste_yaw.exe". Then ENTER ... is that right? Sorry to insist but this is the last adjustment to make for me.

**Comment by simlrh** [comment]
Can you post the output of osvr_print_tree.exe?

**Comment by benybo** [comment]
What do you mean? If I run "osvr_print_tree.exe" in the (bin) file, it opens with lines that scroll quickly, and then closes by itself.

**Comment by simlrh** [comment]
You have to run it from the command line so the output stays there after it's finished

**Comment by benybo** [comment]
Finally ... I find that without using the command line. Just by restarting the computer, this corrects the direction!
But, the head turns very slowly on the right and gives the wrong direction of displacement ...
  Or when I turn on me several times at 360 °, the shift is done faster!? .. to solve the problem, I must
Restart the computer after 20 minutes of use. It does the same with the camera supplied with the HDK2.

**Comment by simlrh** [comment]
Switching the HDK off/on or removing/replacing USB from the PC seems to reset direction somehow as well

**Comment by benybo** [comment]
Yes indeed. Putting the HDK2 on OFF (disconnect the 12v power supply) then ON, restored the sense! .. So the problem is with the Magnetometer of the hdk2 which would be unstable! ... is there a setting for the Magnetometer Of the HDK2? Or the problem would not be resolved yet?
